bad_connector_woocommerce/models/sale_order/common.py

```diff
@@ -42,6 +42,7 @@ class SaleOrder(models.Model):
     is_final_status = fields.Boolean(
         related="woo_order_status_id.is_final_status", string="Final Status"
     )
+    woo_order_status_code = fields.Char(related="woo_order_status_id.code")
```

Comment:
why do we have this at level of order?

Reply:
to hide the button "Export Delivery Status" whenever the sale order is fully returned.